tests/installation/setup_zdup.pm

```diff
@@ -19,8 +19,8 @@ use utils;
 sub run() {
     my ($self) = @_;
 
-    $self->wait_boot(ready_time => 600);
-
+    assert_screen 'displaymanager', 500;
+    ensure_unlocked_desktop;
```

Comment:
This change caused https://progress.opensuse.org/issues/18996, i.e. broke all openSUSE zdup migrations. This is what I meant and I still assume using `wait_boot` is the right approach here. I will see what I can do about it.